readme/clipper.md

```diff
@@ -4,6 +4,18 @@ The Web Clipper is a browser extension that allows you to save web pages and scr
 
 <img src="https://joplinapp.org/images/WebExtensionScreenshot.png" style="max-width: 50%; border: 1px solid gray;">
 
+# Installing the extension
+
+## In Chrome
+
+- Open https://chrome.google.com/webstore/detail/joplin-web-clipper/alofnhikmmkdbbbgpnglcpdollgjjfek
```

Comment:
This link is not valid and results in a 404.

Reply:
Strange, it loads ok for me

Reply:
Oops. My bad. I missed the line break.

Reply:
@tibu please ignore the comment about the 404.

However, I'm still not sure, if we should duplicate info. I'm not a fan of doing so. We duplicate info in the terminal.md file. It has already happened that we update one, but not the other.